### DISCUSSION TITLE
知识库+聊天机器人  机器人的回复里面的“工作流”“引用”这两块可以不展示给用户吗？要在哪里改

### DISCUSSION THREAD

**Discussion by rudadasir** [langgenius/dify#4003] [General]
![Uploading 微信截图_20240430135202.png…]()
知识库+聊天机器人  机器人的回复里面的“工作流”“引用”这两块可以不展示给用户吗？要在哪里改

**Comment by crazywoola** [comment]
Close due to #1542
